EventKit/eventkit-cloud
# Update update_providers script to handle more fixture models and FK links

## Description of Improvement

Improves update_providers script to handle additional fixture models, and create test fixture file supporting additional models.

## How to test

Try to run update_providers script. Should work correctly, and add the following to the db (if included in fixture file):
regions
users
regional policies
attribute classes
topics

## Quality Assurance 

The following items have been updated:
 
## Screenshots

:smile:
